src/Api/getCityWeather.tsx

```diff
@@ -0,0 +1,18 @@
+const baseUrl = "http://api.weatherapi.com";
+const endpointPath = "v1/current.json";
+
+const getUrl = (city: string) => {
+	const searchParams = new URLSearchParams({
+		key: "c6068278a47d4030aad124640241101",
```

Comment:
Same comment as in the previous file.
```suggestion
		key: process.env.WEATHER_API_KEY
```